executable-monitor/action.yml

```diff
@@ -0,0 +1,31 @@
+name: 'executable-monitor'
+description: 'Runs and executable until a termination line is hit or a timeout occurs. Reports if the executable completed successfully or failed.'
+inputs:
+  exe-path:
+    description: 'Path to the executable to run.'
+    required: true
+  log-dir:
+    description: 'Path to directory to store logs.'
+    required: true
+  success-line:
+    description: 'Line of output from executable indicating success.'
+    required: false
+    default: "Demo completed successfully."
```

Comment:
The success line is a variable - you pass it to the action in the workflow "Demo completed successfully." is just the default.

Reply:
Yeah, was thinking it might make sense to use a [variable](https://docs.github.com/en/actions/learn-github-actions/variables) for the default values, since it's declared in so many places. But guess it's nearly just as easy to just find and replace the value if we change the default message.

Reply:
Maybe you misunderstand - from a workflow that uses this action, I can set success-line to whatever is needed by the workflow. e.g. I would use the action then you specify "with: success-line: <your success line>"

Reply:
Oh I understand how the default works, I was just saying that instead of having "Demo completed successfully." as a string literal in 13 places between this file and the PR in the CSDK repo it might make sense to declare the defaults as variables.